vpr/src/base/ShowSetup.cpp

```diff
@@ -553,6 +556,9 @@ static void ShowRouterOpts(const t_router_opts& RouterOpts) {
                 case e_heap_type::BINARY_HEAP:
                     VTR_LOG("BINARY_HEAP\n");
                     break;
+                case e_heap_type::FOUR_ARY_HEAP:
```

Comment:
I suggest refactoring the router option printing to there is a helper function for the common parts of the GLOBAL and DETAILED router types (or you just put both in a common if statement), with only the unique parts of each printed separately.  Most options apply to both, so with a little refactoring we can remove the code duplication.